crates/payload/builder/src/service.rs

```diff
@@ -171,6 +170,14 @@ where
     ) -> Result<PayloadId, PayloadBuilderError> {
         self.send_new_payload(attr).await?
     }
+
+    /// Sends a message to the service to subscribe to payload events.
+    /// Returns a receiver that will receive the payload events.
+    pub fn subscribe(&self) -> oneshot::Receiver<broadcast::Receiver<PayloadEvents<Engine>>> {
```

Comment:
I want a wrapper struct for `broadcast::Receiver` here so we can easily add more functions, for example to get a stream of PayloadAttr